pkg/api/validate/dynamic/quota.go

```diff
@@ -8,16 +8,49 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/Azure/go-autorest/autorest/azure"
-	"github.com/sirupsen/logrus"
-
 	"github.com/Azure/ARO-RP/pkg/api"
-	"github.com/Azure/ARO-RP/pkg/env"
-	"github.com/Azure/ARO-RP/pkg/util/aad"
-	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/compute"
-	"github.com/Azure/ARO-RP/pkg/util/refreshable"
 )
 
+func (dv *dynamic) ValidateQuota(ctx context.Context, oc *api.OpenShiftCluster) error {
+	// return early if this is not the create
+	if oc.Properties.Install == nil || oc.Properties.Install.Phase != api.InstallPhaseBootstrap {
+		return nil
+	}
```

Comment:
I know this is a copy/paste but it doesn't look right any more ...

Reply:
I think we need this if we want to make sure it does not run on other phases?